jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/instance/list/ProcessInstanceSummaryErrorPopoverCell.java

```diff
@@ -0,0 +1,123 @@
+    private Anchor mainAnchor;
+
+    @Inject
+    @DataField("popoverContent")
+    private Span popoverContent;
+
+    @Inject
+    @DataField("contentErrCount")
+    private Span contentErrCount;
+    
+    @Inject
+    @DataField(VIEW_ERROR_LINK_NAME)
+    private Anchor viewErrAnchor;
+
+    @Override
+    public void render(Context context, ProcessInstanceSummary value, SafeHtmlBuilder sb) {
+        int errCount = (value != null ? value.getErrorCount() : 0);
+
+        mainAnchor.setTextContent(Integer.toString(errCount));
+
+        if(errCount > 0){
+            mainAnchor.setAttribute("data-toggle", "popover");
+            mainAnchor.getClassList().add(ERROR_PRESENT_STYLE);
+            viewErrAnchor.setTitle(I18N.ErrorCountViewLink());
+            viewErrAnchor.setTextContent(I18N.ErrorCountViewLink());
+            viewErrAnchor.setAttribute(PROCESS_INSTANCE_ATTRIBUTE, Long.toString(value.getProcessInstanceId()));
+            contentErrCount.setTextContent(I18N.ErrorCountNumber(errCount));
+            mainAnchor.setAttribute("data-content", sanitizeAttributeHtml(popoverContent.getInnerHTML()));
+            
+            Scheduler.get().scheduleDeferred(() -> initPopovers(VIEW_ERROR_LINK_NAME, PROCESS_INSTANCE_ATTRIBUTE));
+        }else{
+            mainAnchor.removeAttribute("data-toggle");
+            mainAnchor.removeAttribute("data-content");
+            mainAnchor.getClassList().remove(ERROR_PRESENT_STYLE);
+        }
+        sb.appendHtmlConstant(mainAnchor.getOuterHTML());
+    }
+    
+    public void openErrorView(String pidStr){
+        Long pid = (pidStr != null ? Long.valueOf(pidStr) : null);
+        PlaceRequest request = new DefaultPlaceRequest(PerspectiveIds.EXECUTION_ERRORS);
+        if(pid != null){
+            //TODO: add process instance ID to PlaceRequest when navigation is implemented. E.g.:
+            //request.addParameter(ErrorManagementPerspective.PROCESS_ID, pid);
+        }
+        placeManager.goTo(request);
+    }
+    
+    private native void initPopovers(String linkName, String procIdAttrName) /*-{
+        var thisCellRef = this;
+        $wnd.jQuery(document).ready(function(){
+            $wnd.jQuery("[data-toggle='popover']")
+                .popover()
+                .off("inserted.bs.popover")
+                .on("inserted.bs.popover", function(){
+                    $wnd.jQuery("[data-field='" + linkName + "']")
+                        .off("click")
+                        .on("click", onViewErrorsClick);                
+                });
+            function onViewErrorsClick(){
+                var processInstId = $wnd.jQuery(this).attr(procIdAttrName);
+                thisCellRef.@org.jbpm.workbench.pr.client.editors.instance.list.ProcessInstanceSummaryErrorPopoverCell::openErrorView(Ljava/lang/String;)(processInstId);
+            }
+        });
+    }-*/;
+    
+    private String sanitizeAttributeHtml(String html){
+        return html.replace("\"",  "'"); 
```

Comment:
@rbarriuso I think data-content can actually be a function, that way you could simply call a method back in the class to get the inner HTML. That way you wouldn't need any manipulation in the html String, right?